PSIni/Private/Remove-EmptyLine.ps1

```diff
@@ -9,14 +9,14 @@
         [Parameter( Mandatory )]
         [ValidateNotNullOrEmpty()]
         [String]
-        $FilePath,
+        $Path,
 
         $Force
     )
 
     $lf = if ((-not $PSVersionTable.Platform) -or ($PSVersionTable.Platform -eq "Win32NT")) { "\r\n" }
     else { "\n" }
 
-    $content = (Get-Content -Path $FilePath -Raw) -replace "(${lf})*$", "" -replace "(${lf}){3,}", ""
-    Set-Content -Value $content -Path $FilePath -Encoding $Encoding -Force:$Force
+    $content = (Get-Content -Path $Path -ErrorAction SilentlyContinue -Raw) -replace "(${lf})*$", "" -replace "(${lf}){3,}", ""
```

Comment:
Why the `SilentlyContinue`?
Should we not fail if the file does not exist?

Reply:
I have moved the `silentlyContinue` from `Export-Ini` as a reminder of this function's (Remove-EmptyLines) existence.

In export it was used Because later on we are recreating/overwriting the file anyway.

Depending on how we want to handle new line cleaning, this function should be either deleted or marked as legacy code.